extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -97,6 +97,22 @@ let BoundPropertyDef;
  */
 let BoundElementDef;
 
+/**
+ * The options bag for binding application.
+ * - skipAmpState: If true, skips <amp-state> elements.
+ * - constrain: If provided, restricts application to children of the provided elements.
+ * - evaluateOnly: If provided, caches the evaluated result on each bound element and skips the actual DOM updates.
+ *
+ * @typedef {Object} ApplyOptionsDef
+ * @property {boolean=} skipAmpState - If true, skips <amp-state> elements.
+ * @property {Array<!Element>=} constrain
+ * @property {boolean=} evaluateOnly
+ *   skipAmpState: boolean=,
+ *   constrain: Array<!Element>=,
+ *   evaluateOnly: boolean=
+ * }}
```

Comment:
Are lines 110-112 supposed to be removed?

Reply:
Definitely 👍